code/modules/research/nodes/bluespace.dm

```diff
@@ -112,7 +112,7 @@
 
 /datum/technology/bluespace_shield
 	name = "Bluespace Shields"
-	desc = "Improved light-shields with using of bluespace technologies."
+	desc = "Improved light-shields based on bluespace technologies."
```

Comment:
Here means that we have nasic light shields and we improve it with using of bluespace.